data/cloudsek.mdx

```diff
@@ -0,0 +1,19 @@
+---
+name: CloudSEK
+description: At CloudSEK we combine the power of Cyber Crime monitoring, Brand Monitoring, Attack Surface monitoring, and Supply chain intelligence to give context to our customers’ digital risks. Our single unified dashboard allows customers to triage and visualize all digital threats in one place. We also offer workflows and integrations to manage andremediate the identified threats.
+founded: 2015
+based: Both
+domain: Cyber Security
+logo: cloudsek.png
+location: Bengaluru, Karnataka, India
```

Comment:
The company is in Banglore 🤔. Then how's this a startup in kerala?

Reply:
Is there a home office in kerala?

Reply:
https://www.tofler.in/cloudsek-information-security-private-limited/company/U72900KL2015PTC039177

They have a registered office in Ernakulam

Reply:
Ok done 😊